### ISSUE TITLE
Gray screen on login due to Freesync?

### ISSUE THREAD

**Issue by Miro-Andrin** [pop-os/pop#1185]
**Distribution (run `cat /etc/os-release`):**
 NAME="Pop!_OS"
VERSION="20.04 LTS"
  
**Related Application and/or Package Version (run `apt policy $PACKAGE NAME`):**
None that i know the name of, it  would be program responsible for the login screen. 

**Issue/Bug Description:**
Today i got a [new wide screen computer](https://www.komplett.no/product/1148694/datautstyr/skjermer/skjermer/svive-35-curved-skjerm-pyx-35c601), with Freesync. I have a Radeon RX 5500 XT GPU. After installing the new screen, i tried starting my existing pop-os install. This resulted in a gray screen. 

I have stumbled into a solution for my problem, but its quite annoying.  Once pop-os launches to the gray screen, i need to turn off freesync on my monitor. This makes the login fields appear. Once i have entered my password the screen hangs again. I then need to re-enable freesync on the monitor. Then the desktop appears. 

**Other Notes:**
I also tried re installing pop-os into a different drive. The installation worked, but it had the same problem once installed.

**Comment by faedy2** [comment]
I wonder...
When you get the gray screen, can you please try these steps to see if you can get to the login screen?

1. Tap '3' on numpad
2. Hit 'CTRL+ALT+F4'
3. Hit 'CTRL+ALT+F1'

**Comment by Miro-Andrin** [comment]
> I wonder...
> When you get the gray screen, can you please try these steps to see if you can get to the login screen?
> 
> 1. Tap '3' on numpad
> 2. Hit 'CTRL+ALT+F4'
> 3. Hit 'CTRL+ALT+F1'

I don't have a keyboard with a numpad. It might take one or two days before i can procure one.

**Comment by Miro-Andrin** [comment]
@faedy2  I have tested the steps you suggested, and nothing happend. I have noticed some other things. 
1) i dont nede to cycle freesync to exit the gray screen. Turning the monitor off and on does the trick. 

2) When i used my laptop (same os) plugged  in the same hdmi,  everything worked. 

3) My monitor supports hdmi 2.0 and I used that port originally. If I change to one of the  hdmi 1.4 ports, then I get a mouse pointer, but still just the gray background. When trying your steps on this port, then the mouse disappeared on step 2 ( alt +ctrl +f4). 

What type of information would be helpful?

**Comment by faedy2** [comment]
Yeah your mouse should disappear on (Alt+Ctrl+f4), but then should come
back when you do (alt+ctrl+f1)
For me, I get a gray background with nothing else, not even a cursor
Not too sure what next steps we can take sadly :(
